msteams-platform/graph-api/rsc/resource-specific-consent.md

```diff
@@ -144,6 +200,17 @@ Once the app has been installed to a team, you can use [Graph Explorer](https://
 > - Make a **GET** call to the following endpoint: `https://graph.microsoft.com/beta/groups/{teamGroupId}/permissionGrants`. The clientAppId field in the response will map to the appId specified in the Teams app manifest.
   ![Graph explorer response to GET call.](../../assets/images/graph-permissions.png)
 
```

Comment:
Can we also add a subtle crossreference: 
see also over here - point to listing details of apps installed in a team.